components/stage/scripts/create.py

```diff
@@ -93,6 +107,17 @@ def _install_stage():
             shell=True)
 
 
+def configure_script(script_name, description):
+    utils.deploy_sudo_command_script(
+        script_name,
+        description,
+        component=SERVICE_NAME,
+        allow_as=STAGE_USER,
+    )
+    utils.chmod('a+rx', '/opt/cloudify/stage/' + script_name)
+    utils.sudo(['usermod', '-aG', utils.CLOUDIFY_GROUP, STAGE_USER])
```

Comment:
sure we want to call this for every script?

Reply:
We do for every script that currently uses this, yes.